.swm/VW4rf3AxRslfT7lwaug7.swm

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "VW4rf3AxRslfT7lwaug7",
+    "name": "Implement a new PBA —  `ScheduleJobs`",
```

Comment:
don't we already have this PBA? Maybe we should instead start with a more trivial task, like a PBA that adds a text tile to temp dir?

Reply:
Maybe we can add more playlists/units for trivial tasks; this one was used in the OpenSauced presentation so we should probably keep this separate.

Reply:
Actually, this doesn't require the swimmer to add the actual job scheduling commands. They're only required to fetch the commands from an already existing function, `get_commands_to_schedule_jobs`, which is mentioned in the unit's description. So it's not really hard.